src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -68,10 +68,11 @@ class ViewClippingWindow final : public Window
 {
 private:
     CoordsXY _selectionStart;
-    CoordsXY _previousClipSelectionA;
-    CoordsXY _previousClipSelectionB;
+    CoordsXY _previousClipSelectionA = { 0, 0 };
+    CoordsXY _previousClipSelectionB = { gMapSize.x * 32, gMapSize.y * 32 };
```

Comment:
```suggestion
    CoordsXY _previousClipSelectionB = gMapSize.ToCoordsXY();
```

Same for the other occurrences.

We are moving away from manually multiplying coords, especially by naked numbers.